web_src/js/modules/fetch.js

```diff
@@ -14,6 +14,7 @@ function request(url, {headers, data, body, ...other} = {}) {
     } else if (data instanceof URLSearchParams) {
       contentType = 'application/x-www-form-urlencoded';
       body = data;
```

Comment:
Should we perhaps hardcode the content type to `charset=utf-8` in this method?
I don't see a reason why we shouldn't set it.

Reply:
According to https://stackoverflow.com/a/16829056/808699, utf-8 is the default.

Reply:
Yes, and W3 disagrees:
> It is very important to always label Web documents explicitly. HTTP 1.1 says that the default charset is ISO-8859-1.

https://www.w3.org/International/articles/http-charset/index.en.html

Reply:
I trust WHATWG more than I do W3.

https://url.spec.whatwg.org/#urlencoded-parsing

Reply:
See https://stackoverflow.com/questions/6825713 for more info. W3C at this point is not really that much of a trustworthy source. WHATWG and to lesser extend MDN is much more trustworthy.